.github/ISSUE_TEMPLATE/Add Conference.yaml

```diff
@@ -56,7 +56,7 @@ body:
     attributes:
       label: CFP Dates
       description: The start and end date of the Call for Papers.
-      placeholder: "YYYY-MM-DD"
+      placeholder: "DD (MMM) -DD MMM YYYY"
```

Comment:
Apologies on this one in my description. This placeholder should just be <DD MMM YYYY> as #24 indicates that we're only tracking the CFP End Date as we usually announce the event during the Open CFP Call.

If you want to update the code to correct #24 and this issue, that would be awesome and we can go ahead and push this.